libethereum/BlockQueue.cpp

```diff
@@ -179,6 +167,21 @@ void BlockQueue::verifierBody()
 	}
 }
 
+void BlockQueue::drainVerified()
```

Comment:
functions that operate on the assumption there is one or more mutexes already locked should include that in their name. e.g. `drainVerified_WITH_BOTH_LOCKS`